src/data/Calendar.js

```diff
@@ -3,24 +3,44 @@ export const labels = {
     color: "yellow",
     background: "!bg-hackathon-green-200",
   },
-  operations: {
-    color: "yellow",
+  director: {
+    color: "pink",
     background: "!bg-hackathon-green-200",
   },
-  webdev: {
+  marketing: {
     color: "green",
     background: "!bg-hackathon-green-200",
   },
-  marketing: {
+  operations: {
     color: "red",
     background: "!bg-hackathon-green-200",
   },
   sponsorship: {
+    color: "lightgreen",
+    background: "!bg-hackathon-green-200",
+  },
+  recruitment: {
+    color: "purple",
```

Comment:
color and background should be corresponding colors, we have purple and green which is a mismatch